src/components/Chat/InputMessageV2.tsx

```diff
@@ -0,0 +1,207 @@
+import {Alert, Image, StyleSheet, Text, TextInput, TouchableOpacity, View} from 'react-native';
+
+import IconSend from '../../assets/icon/IconSendComment';
+import {colors} from '../../utils/colors';
+import dimen from '../../utils/dimen';
+import {normalizeFontSize} from '../../utils/fonts';
+import ToggleSwitch from '../ToggleSwitch';
+import SheetEmoji from './SheetEmoji';
+
+const styles = StyleSheet.create({
+  main: {
+    flex: 1,
+    flexDirection: 'row',
+    alignItems: 'flex-end'
+  },
+  container: {
+    flex: 1,
+    backgroundColor: colors.white,
+    paddingHorizontal: dimen.normalizeDimen(6)
+  },
+  btnEmoji: {
+    width: dimen.normalizeDimen(24),
+    height: dimen.normalizeDimen(24),
+    borderRadius: dimen.normalizeDimen(24 / 2),
+    justifyContent: 'center',
+    alignItems: 'center',
+    flexShrink: 0,
+    marginBottom: dimen.normalizeDimen(4)
+  },
+  icSendButton: {
+    alignSelf: 'center'
+  },
+  containerInput: {
+    flex: 1,
+    backgroundColor: colors.lightgrey,
+    flexDirection: 'row',
+    alignItems: 'flex-end',
+    paddingVertical: dimen.normalizeDimen(6),
+    paddingHorizontal: dimen.normalizeDimen(8),
+    borderRadius: dimen.normalizeDimen(12),
+    minHeight: dimen.normalizeDimen(32),
+    marginHorizontal: dimen.normalizeDimen(6)
+  },
+  input: {
+    flex: 1,
+    backgroundColor: colors.lightgrey,
+    color: colors.black,
+    fontFamily: 'Inter',
+    fontSize: normalizeFontSize(14),
+    fontStyle: 'normal',
+    fontWeight: '400',
+    marginRight: dimen.normalizeDimen(12),
+    minHeight: dimen.normalizeDimen(20),
+    maxHeight: dimen.normalizeDimen(80),
+    textAlignVertical: 'center',
+    paddingTop: dimen.normalizeDimen(2)
+  },
+  btn: {
+    borderRadius: dimen.normalizeDimen(32 / 2),
+    width: dimen.normalizeDimen(32),
+    height: dimen.normalizeDimen(32),
+    display: 'flex'
+  },
+  enableButton: {
+    backgroundColor: colors.bondi_blue
+  },
+  disableButton: {
+    backgroundColor: colors.gray1
+  },
+  previewPhotoContainer: {
+    marginTop: dimen.normalizeDimen(5),
+    marginBottom: dimen.normalizeDimen(5)
+  },
+  imageStyle: {
+    height: dimen.normalizeDimen(64),
+    width: dimen.normalizeDimen(64),
+    marginRight: dimen.normalizeDimen(10)
+  },
+  containerDelete: {
+    position: 'absolute',
+    top: 0,
+    right: 0,
+    height: dimen.normalizeDimen(25),
+    width: dimen.normalizeDimen(25),
+    backgroundColor: colors.holytosca,
+    alignItems: 'center',
+    justifyContent: 'center',
+    zIndex: dimen.normalizeDimen(100),
+    borderRadius: dimen.normalizeDimen(13)
+  },
+  labelToggle: {
+    fontSize: dimen.normalizeDimen(8),
+    fontWeight: '400'
+  }
+});
+
+export interface InputMessageV2Props {
+  onSendButtonClicked: (message: string) => void;
+  type: 'SIGNED' | 'ANONYMOUS';
+  emojiCode?: string;
+  emojiColor?: string;
+  username?: string;
+  profileImage?: string;
+  onToggleConfirm?: () => void;
+  isShowToggle?: boolean;
+}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `InputMessageV2Props` interface could be improved by adding comments to describe the purpose of each prop. This will make it easier for other developers to understand what each prop is used for.



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
tidak perlu